src/test/java/homeWorks/ttykhovetska/JUnitTestNGHomeWork.java

```diff
@@ -0,0 +1,92 @@
+package homeWorks.ttykhovetska;
+
+import atqc.junit_testng.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class JUnitTestNGHomeWork {
+
+    public String baseUrl="https://try.test.io/test-io-platform/";
+    public String driverPath = "/opt/homebrew/bin/chromedriver";
+    public WebDriver driver;
+    private static final String User_First_Name = "David";
+    private static final String User_Last_Name = "Dakhovny";
+    private static final String User_Company = "Marsik";
+    private static final String User_EMail = "moonlight@gmail.com";
```

Comment:
your test mail is invalid. According to the pattern in element mail should be like "firstname.lastname@example.com"